### PR TITLE
Task a-tiny-js-world

### PR DESCRIPTION
**Pull request by BlueLamantine**
### A Tiny JS World

[Demo](https://bluelamantine.github.io//a-tiny-JS-world/) |
[Code base](https://github.com/BlueLamantine/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/BlueLamantine/a-tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+   legs: 2,
+   hands: 2,
+   saying: cat.saying,
+   friends: 'No friends'
+};
+const jsWorld = [man,woman,dog,cat,catWoman];
+// ======== OUTPUT ========
+jsWorld.forEach( el => {
+   print(Object.values(el).join(';'));
+});
```
Comment (MaxKalashnyk) [inline submissions/BlueLamantine/a-tiny-js-world/index.js:58]: Add an empty line here, please

File: submissions/BlueLamantine/a-tiny-js-world/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+   species: 'Supernatural',
+   name: 'Mary',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: cat.saying,
+   friends: 'No friends'
+};
+const jsWorld = [man,woman,dog,cat,catWoman];
+// ======== OUTPUT ========
```
Comment (MaxKalashnyk) [inline submissions/BlueLamantine/a-tiny-js-world/index.js:55]: I think this comment is redundant here. To my mind, it's clear and it doesn't need to be commented

**Review by MaxKalashnyk — CHANGES_REQUESTED** [review]
Nice work! Look at my comments, please

File: submissions/BlueLamantine/a-tiny-js-world/index.js (hunk 1 above)
Comment (BlueLamantine) [reply submissions/BlueLamantine/a-tiny-js-world/index.js]: ok

File: submissions/BlueLamantine/a-tiny-js-world/index.js (hunk 2 above)
Comment (BlueLamantine) [reply submissions/BlueLamantine/a-tiny-js-world/index.js]: ok

**Comment by BlueLamantine** [comment]
@MaxKalashnyk  thanks for review! I followed your recommendations.

**Review by MaxKalashnyk — APPROVED** [review]
Good job!
Sorry for the delay with approving.

**Pull request merged**